### PR TITLE
5.6.4 release notes

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

**Review by MichaelBaj — APPROVED** [review]

File: docs/release_notes_128t_5.6.md [hunk 1]
```diff
@@ -24,6 +24,37 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+
+### Resolved Issues
+
+- **I95-48076 SSR Failover on GRE tunnels not working:** The base interface giid is now used to identify the state of a GRE tunnel next-hop.
+------
+- **I95-48158 Unable to capture child services using session capture:** When a session capture is configured on a child service (e.g., `social.internet` instead of `internet`), the session is now recorded.
+------
+- **I95-48427 BGP ignoring multihop TTL (Time To Live) setting leading to invalid nexthop:** Resolved an issue where BGP may temporarily "forget" about the TTL value configured for a neighbor.
+------
+- **I95-48507 VLAN packets are generated without a valid VLAN from the flow-move cache:** Resolved an issue where sessions could be modified incorrectly when a VLAN is present and session resiliency is enabled for failover.
```
Comment (jeffmart-jnpr) [inline docs/release_notes_128t_5.6.md:48]: This fix actually went in 5.6.3, can this be moved to the 5.6.3 section?

File: docs/release_notes_128t_5.6.md [hunk 2]
```diff
@@ -24,6 +24,37 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-48508 Keep-alive cache may cause worker core CPU spikes:** Resolved potential worker core utilization CPU spikes by utilizing aggressive keep-alive timeouts.
+------
+- **I95-48600 Compare Session ID's to prevent flow collisions:** Re-use of sessions is prevented when waypoint pool is exhausted and sessions linger on egress router.
+------
+- **I95-48685 Infinite Proxying Between HA systems:** Updates made to prevent the recursive proxying that occurs between two HA systems when one of the upstream targets is unavailable.
```
Comment (jeffmart-jnpr) [inline docs/release_notes_128t_5.6.md:54]: Looks like for the 5.5 release notes this is documented different, this PR should align the definition of the jira for both releases to provide more information on how the customer could observe the issue.

**Review by jeffmart-jnpr — CHANGES_REQUESTED** [review]
Please update based on individual comments

File: docs/release_notes_128t_5.6.md (hunk 2 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.6.md]: corrected.

File: docs/release_notes_128t_5.6.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.6.md]: moved

**Review by MichaelBaj — APPROVED** [review]

**Review by jeffmart-jnpr — APPROVED** [review]

**Pull request merged**